api/src/test/java/org/openmrs/obs/AbstractHandlerTest.java

```diff
@@ -43,6 +44,8 @@ public class AbstractHandlerTest extends BaseContextSensitiveTest {
 	private final String FILENAME = "mytxtfile.txt";
 	
 	private  AbstractHandler handler;
+
+	private Obs obs;
```

Comment:
It's better not to introduce a class-level variable in tests where it can be avoided.